compositor_integration_tests/src/test_runner.rs

```diff
@@ -0,0 +1,22 @@
+use std::env;
+
+use compositor_integration_tests::integration_tests;
+use compositor_render::use_global_wgpu_ctx;
+use libtest_mimic::{Arguments, Trial};
+use video_compositor::logger;
+
+pub fn main() {
+    env::set_var("LIVE_COMPOSITOR_WEB_RENDERER_ENABLE", "0");
+    ffmpeg_next::format::network::init();
+    logger::init_logger();
+
+    use_global_wgpu_ctx();
+
+    let args = Arguments::from_args();
+    let tests = integration_tests()
+        .into_iter()
+        .map(|test| Trial::test(test.name, move || test.run_test().map_err(Into::into)))
+        .collect();
+
+    libtest_mimic::run(&args, tests).exit();
```

Comment:
It's a custom test harness which makes each integration test behave like a normal test:
- `cargo test -- --list` - lists every integration test as a separate test
- It's compatible with `cargo test` and `cargo nextest`
- Each test is run in parallel unless tests are run with `--test-threads=1` flag

Reply:
Also is there a reason why you are introducing new lib instead of just using regular rust test?

Reply:
Sorry, I forgot to commit and push the latest changes. I removed `libtest-mimic` in favor of normal tests.
The problem I had with normal tests was that you can't call test functions like normal functions. You'd have to define a wrapper function for each test. For that reason, I decided to use that lib. It would remove some boilerplate from test definition.

In the end, I decided to use normal tests because they are more readable since there's less 'magic' involved.

Please check the latest commit.